mix.exs

```diff
@@ -19,7 +19,8 @@ defmodule ChatApp.MixProject do
   def application do
     [
       mod: {ChatApp.Application, []},
-      extra_applications: [:logger, :runtime_tools]
+      extra_applications: [:logger, :runtime_tools],
+      start_phases: [tables: []]
```

Comment:
Let's rename it to `initialize_ets_tables` to be more clear about what is actually happening.
These phases are **actions**; please name them accordingly.